### PR TITLE
Add Handsontable to the registry and package-overrides.

### PR DESCRIPTION
**Pull request by jansiegel**

### CODE REVIEW

File: package-overrides/github/handsontable/handsontable@0.24.1.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "main": "dist/handsontable.full.min",
```
Comment (guybedford) [inline package-overrides/github/handsontable/handsontable@0.24.1.json]: It's best to use the unminified dependency for best debugging support.

**Comment by guybedford** [comment]
Is anything from the discussion in https://github.com/jspm/registry/issues/697 relevant here? That seemed to include dependencies on zeroclipboard, pikaday, moment and jquery. Are any of those needed by Handsontable for the default import?

**Comment by jansiegel** [comment]
> Is anything from the discussion in #697 relevant here? That seemed to include dependencies on zeroclipboard, pikaday, moment and jquery. Are any of those needed by Handsontable for the default import?

I think it should work fine. Our main dependencies (pikaday, moment.js and ZeroClipboard) are all included in the `full` build - maybe except for the ZeroClipboard's `.swf` file, but I don't think there's a way around that. As for the jQuery, since about a year ago we're jQuery-free, so it shouldn't be and issue.

As for the RuleJS library, it's not really a Handsontable plugin - rather a separate, currently not fully supported repository, so I wouldn't treat it as an integral part of the Handsontable package.

File: package-overrides/github/handsontable/handsontable@0.24.1.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "main": "dist/handsontable.full",
+  "dependencies": {
+    "css": "0.1.0"
```
Comment (guybedford) [inline package-overrides/github/handsontable/handsontable@0.24.1.json]: `"css": "*"` is advisable here.

**Comment by guybedford** [comment]
Ok that all looks great to go then pending the css comment.

Also once this is merged, can also be advisable to include this configuration under `"jspm"` in the Handsontable package.json itself so that can become the source of truth but no pressure here at all.

**Comment by jansiegel** [comment]
> Also once this is merged, can also be advisable to include this configuration under "jspm" in the Handsontable package.json

Yup, it's probably a good idea, we'll take it into consideration with our next releases :)

**Comment by guybedford** [comment]
:+1:

**Comment by ineentho** [comment]
This does not work with jspm .12, there are several issues:
- The css file points to the wrong path (changing it to ./handsontable.full.css works, so no big deal)
- Something with traceur $defineProperty not being defined.

It works when importing with System.import('handsontable'), but not within a esm module. My guess is there is some conflict between babel and traceur, but I didn't figure out what was wrong.

Test case is a clean jspm project:

```
mkdir httest && cd httest && npm init -y && npm i -S jspm@beta && jspm init -y && jspm i handsontable
echo '
<script src="jspm_packages/system.src.js"></script>
<script src="jspm.browser.js"></script>
<script src="jspm.config.js"></script>
<script>
    System.import("httest")
</script>
' >> index.html
echo "import 'handsontable'" >> src/httest.js
```

I did some digging, but I know very little about how the handsontable bundle is made.

**Pull request merged**